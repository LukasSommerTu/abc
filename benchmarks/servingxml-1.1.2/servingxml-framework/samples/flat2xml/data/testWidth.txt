TX#00000000010016This is a remarkXXXXXXXXXX
TX#0000000002              
TX#0000000003    YYYYYYYYYY
TX#00000000030004TestZZZZZZZZZZ