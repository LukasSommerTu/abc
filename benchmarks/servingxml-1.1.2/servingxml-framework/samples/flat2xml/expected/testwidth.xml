<?xml version="1.0" encoding="UTF-8"?>
<Doc>
    <Transaction>
        <Reference>0000000001</Reference>
        <Remarks2>This is a remark</Remarks2>
        <FollowingField>XXXXXXXXXX</FollowingField>
    </Transaction>
    <Transaction>
        <Reference>0000000002</Reference>
        <Remarks2/>
        <FollowingField/>
    </Transaction>
    <Transaction>
        <Reference>0000000003</Reference>
        <Remarks2/>
        <FollowingField>YYYYYYYYYY</FollowingField>
    </Transaction>
    <Transaction>
        <Reference>0000000003</Reference>
        <Remarks2>Test</Remarks2>
        <FollowingField>ZZZZZZZZZZ</FollowingField>
    </Transaction>
</Doc>
